traces

```diff
@@ -75,6 +75,11 @@ const REPOSITORIES_CATEGORIES = [
         'PrestaShop/childtheme-example',
         'PrestaShop/module-companion',
     ],
+    'themes' => [
+        'PrestaShop/classic-theme',
+        'PrestaShop/theme-refacto',
+        'PrestaShop/community-theme-16',
```

Comment:
Wut? Is this repo maintained?

Idk if we want to keep the tracing on this one so I'll wait for another review!

Reply:
Is maybe not maintened, but it's inside the list of activity trace, as contributors have work on it (like PrestaShop 1.4 :D)